guides/common/modules/proc_creating-an-activation-key.adoc

```diff
@@ -44,18 +34,6 @@ endif::[]
 --content-view "_Stack_" \
 --organization "_My_Organization_"
 ----
-. Optional: For {RHEL} 8 hosts, enter the following command to configure the activation key with system purpose to set on hosts during registration to enhance subscriptions auto attachment.
-+
-[options="nowrap", subs="+quotes,verbatim,attributes"]
-----
-# hammer activation-key update \
---organization "_My_Organization_" \
---name "_My_Activation_Key_" \
---service-level "_Standard_" \
---purpose-usage "_Development/Test_" \
---purpose-role "_{RHELServer}_" \
---purpose-addons "_addons_"
-----
```

Comment:
This should be left in. The only part that's no longer accurate is "to enhance subscriptions auto attachment."

Also, system purpose is not limited to RHEL8.

Perhaps
```
Optional: Enter the following command to configure the activation key with system purpose attributes to set on hosts during registration.
This can help determine which repositories are available on the host, and also helps with reporting in the Subscriptions service of the Red Hat Hybrid Cloud Console.
```